src/content/advanced/gitops/apps/update_appcr.md

```diff
@@ -53,7 +53,7 @@ export APP_NAME=APP_NAME
     cd management-clusters/${MC_NAME}/organizations/${ORG_NAME}/workload-clusters/${WC_NAME}/mapi/apps/${APP_NAME}
     ```
 
-1. Edit the `configmap.yaml` if you want to update a non-secret user configuration
+2. Edit the `configmap.yaml` if you want to update a non-secret user configuration
```

Comment:
I think using `1.` for each line is default markdown, doesn't it work?

Reply:
But anything works for me honestly

Reply:
It should work but our CI complains.